### PR TITLE
feat(website/e2e-test): add smoke tests

### PR DESCRIPTION
**Pull request by raffomartini**
Note: missing robot.txt

### CODE REVIEW

File: website/e2e-test/smoke-test.spec.js [hunk 1]
```diff
@@ -32,3 +32,35 @@ const assertUrlLoads = async (page, url) => {
...
+test('Renders login', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/login');
+});
+
+test('Renders logout', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/logout');
+});
+
+test('Renders donate', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/donate?campaign_id=59d32e9805fc4d3388db');
```
Comment (ace-n) [inline website/e2e-test/smoke-test.spec.js:45]: Should we move the `campaign_id` value to a separate variable?

e.g:
```
  const CAMPAIGN_ID = '59d32e9805fc4d3388db';
  await assertUrlLoads(page, EMBLEM_URL + `/donate?campaign_id=${CAMPAIGN_ID}`);
```

File: website/e2e-test/smoke-test.spec.js [hunk 2]
```diff
@@ -32,3 +32,35 @@ const assertUrlLoads = async (page, url) => {
...
+
+test('Renders createCampaign', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/createCampaign');
+});
+
+test('Renders viewDonation', async ({ page }) => {
+  await assertUrlLoads(page, EMBLEM_URL + '/viewDonation?donation_id=f5ea984abf29497bbed7');
+});
+
+// robots.txt - not working right now
```
Comment (ace-n) [inline website/e2e-test/smoke-test.spec.js:60]: Add a link to [this issue](https://github.com/GoogleCloudPlatform/emblem/issues/389) in a comment?
```
// See https://github.com/GoogleCloudPlatform/emblem/issues/389
```

**Review by ace-n — APPROVED** [review]
Thanks for doing this! A few minor nits. 🙂

File: website/e2e-test/smoke-test.spec.js (hunk 2 above)
Comment (raffomartini) [reply website/e2e-test/smoke-test.spec.js:68]: Good idea, done.

File: website/e2e-test/smoke-test.spec.js (hunk 1 above)
Comment (raffomartini) [reply website/e2e-test/smoke-test.spec.js]: Done

**Comment by ace-n** [comment]
@engelke we don't have an instance to point this at (yet), but I'm going to go ahead and merge this PR.

@raffomartini thanks again for the contribution!

**Pull request merged**